libraries/tools/kotlin-gradle-plugin-api/src/main/kotlin/org/jetbrains/kotlin/gradle/dsl/KaptExtensionApi.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.gradle.dsl
+
+import groovy.lang.Closure
+
+/** KAPT extension used to configure stub generation and annotation processing. */
+interface KaptExtensionApi {
```

Comment:
As it becomes official plugin API - could you describe this interface properties via KDoc?

Reply:
Done in 911a97e.